common/src/main/cpp/classes/openxr_fb_passthrough_geometry.cpp

```diff
@@ -91,6 +121,36 @@ void OpenXRFbPassthroughGeometry::destroy_passthrough_geometry() {
 		OpenXRFbPassthroughExtensionWrapper::get_singleton()->destroy_geometry_instance(geometry_instance);
 		geometry_instance = XR_NULL_HANDLE;
 	}
+
+	if (opaque_mesh != nullptr) {
+		opaque_mesh->queue_free();
```

Comment:
Should the opaque mesh be removed from the tree when it's destroyed?

Reply:
I don't see why not, I've added `remove_child(opaque_mesh)` in my latest push to the instances where it's destroyed.